contractcourt/channel_arbitrator.go

```diff
@@ -2579,8 +2627,22 @@ func (c *ChannelArbitrator) resolveHTLCsImmediately(
 		// If we can fail an HTLC immediately (an outgoing HTLC with no
 		// contract and it was not canceled before), then we'll assemble
 		// an HTLC fail packet to send.
+		//
+		// NOTE: In case we initiated the force close locally all the
```

Comment:
I think by the time we hit here we should cancel all the dust HTLCs, for all possible scenarios. This is because the channel always starts at state `StateDefault`, then we advance it.

Reply:
Ohh true did not think about this, but you are absolutely right.